<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Smita-Lenovo" failures="1" tests="3" name="Test1" time="10.368" errors="0" timestamp="2023-03-30T00:36:02 BST">
  <testcase classname="testPackage.LoginPageTestCases" name="VerifyLoginPageWithInvalidUsername_InvalidPassword" time="1.316">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testPackage.LoginPageTestCases.VerifyLoginPageWithInvalidUsername_InvalidPassword(LoginPageTestCases.java:107)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginPageWithInvalidUsername_InvalidPassword -->
  <testcase classname="testPackage.LoginPageTestCases" name="VerifyLoginPageWithValidData" time="1.01"/>
  <testcase classname="testPackage.LoginPageTestCases" name="verifyAdminPage" time="0.002"/>
  <testcase classname="testPackage.BaseClass" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="testPackage.BaseClass" name="@BeforeTest beforeTest" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
</testsuite> <!-- Test1 -->
